src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -29,19 +29,23 @@
 using Microsoft.EntityFrameworkCore.Storage;
 using System.Collections.Generic;
 using System.Diagnostics;
+using JetBrains.Annotations;
 using Microsoft.EntityFrameworkCore.ChangeTracking;
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
 {
     /// <summary>
     /// Maps PostgreSQL arrays to <see cref="List{T}"/>.
     /// </summary>
+    /// <remarks>
+    /// Note that mapping PostgreSQL arrays to .NET arrays is also supported via <see cref="NpgsqlArrayTypeMapping"/>.
```

Comment:
Link to the PostgreSQL docs on arrays, as with some of the other type mapping classes?